pkg/core/util.go

```diff
@@ -25,6 +25,36 @@ func CreateGenesisBlock(cfg config.ProtocolConfiguration) (*block.Block, error)
 		return nil, err
 	}
 
+	txs := []*transaction.Transaction{}
+	for i, s := range cfg.GenesisExtensions.Transactions {
+		signers := make([]transaction.Signer, len(s.Signers)+1)
+		signers[0] = transaction.Signer{
+			Account: nextConsensus,
+			Scopes:  transaction.None, // Enough to pay for the transaction.
+		}
+		for j := range signers[1:] {
+			signers[j+1] = transaction.Signer{
+				Account: s.Signers[j],
```

Comment:
Ah, I see you're using a different sender. But I still think we can limit this to a single signer.

Reply:
Yep, sender is always validator multisig, because it owns all GAS and is able to pay. And the rest of the signers are provided via configuration.

Reply:
I'd simplify this to validators and committee. We can extend (or drop!) this thing if need be.

Reply:
Fixed.